components/NavigationDrawer/tests/unit/MDCNavigationDrawerPresentationControllerTests.m

```diff
@@ -115,4 +115,45 @@ - (void)testSettingShouldAlwaysExpandHeader {
   XCTAssertTrue(
       presentationController.bottomDrawerContainerViewController.shouldAlwaysExpandHeader);
 }
+
+- (void)testShouldIncludeSafeAreaInContentHeight {
```

Comment:
this test looks good, the bottom one looks like a duplicate? Would we be able to test the propagation from the MDCBottomDrawerViewController to the MDCBottomDrawerPresentationController?

Reply:
So this test is actually just a bit of a bonus test (could be added in a separate PR but seemed relevant here or I can remove as well) which tests the existing `shouldIncludeSafeAreaInContentHeight` flag for content height taking safe areas into account. 


The test below does the actual testing around `shouldIncludeSafeAreaInInitialDrawerHeight`, which is the newly added but similarly named flag.

Testing though that the flag propagates from MDCBottomDrawerViewController to the MDCBottomDrawerPresentationController sounds good to me.

I'm signing off for today but will revisit this on Monday and I can modify both of these tests to set the flag on the MDCBottomDrawerViewController and verify that it propagates to both the MDCBottomDrawerPresentationController and the container VC?

Reply:
ohh i see now! Perfect! Yes sounds good to me :)

Reply:
Awesome, thanks @yarneo! I'll make those tweaks to the propagation test then as well as cleaning up the clang-format warning on Monday. Have a great weekend 😄